docs/create-your-own-modern-controllers.md

```diff
@@ -35,7 +35,37 @@ Note that you must return a `Response` object, but this can be a `JsonResponse`
 
 > This controller is the same than the ones used in Back Office. 
 
-Now we have created your controller, you need to declare a route. A route map an action of your controller to an URI.
+You must enable the autoloading for this Controller. For example using a `composer.json` file for your module.
+
```

Comment:
I'm not sure that composer is required, but I'm not enough confident in PrestaShop native autoloader to provide an example: in the end, everything will be managed by Composer I guess :pray: